### ISSUE TITLE
Missing Readme.md

### ISSUE THREAD

**Issue by Dean-Coakley** [davej/CorkJS#17]
A really short, simple readme could be added to explain
* What this project does
* Why it was created
* Contributors guide/How to get involved, including useful links, etc

**Comment by davej** [comment]
Sounds good to me. Feel free to take the lead on this if you want, I probably won't get around to it anytime soon. :)

**Comment by Dean-Coakley** [comment]
@davej Yeah, I'll get around to it when I can. Just wanted to check if you had some extra ideas of stuff to be added. For example I somewhat wonder if contrib guide should be added as we have almost no guidelines/requirements. It seems we don't mind about branches, commit messages(within reason), etc

**Comment by Dean-Coakley** [comment]
Ehh.. I guess now with linting and to try stop people adding libraries.

**Comment by davej** [comment]
Yeah. There are no guidelines really but feel free to add them.

I don't really mind want they are. The idea behind no libraries is just to keep the repo relatively simple so that newcomers can come along and make changes or add pages easily.
